src/Pages/SignUpPage/SignUpPage.tsx

```diff
@@ -22,7 +35,7 @@ const SignUpPage: React.FC = () => {
                 <TextField
                     required
                     label="Name"
```

Comment:
@rooshnie23 Don't think we need a name field in the Signup page. In the backend we only have the username and the password when signing up. I don't think we need to include it. It is not being used anywhere in the application too.

Reply:
Definitely, I'll bring the change to it.

Reply:
Done!